go-ethereum-on-ubuntu/README.md

```diff
@@ -0,0 +1,223 @@
+# Geth Private Node
+
+This Microsoft Azure template deploys a single Ethereum client with a private chain for development and testing.
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://azuredeploy.net/)
```

Comment:
This url code needs to be updated. See the button in the other template https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-quickstart-templates%2Fmaster%2Fblockapps-strato%2Fazuredeploy.json